src/transformers/models/mctc/modeling_mctc.py

```diff
@@ -0,0 +1,904 @@
+# coding=utf-8
+# Copyright 2022 Chan Woo Kim The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2022 Meta AI and The HuggingFace Inc. team. All rights reserved.
```

Reply:
Sorry we sadly cannot add individual contributors to the Copyright here if I remember correctly cc @sgugger

Reply:
Yes, we use organizations or "the Xxx paper authors" usually.

Reply:
Oh haha right that's embarrassing I think that was added automatically when I used the `transformers-cli add-new-model` command.